### DISCUSSION TITLE
Is it possible to a read a api output in postgen and use it in the file context ?

### DISCUSSION THREAD

**Discussion by kabilanmani93** [cookiecutter/cookiecutter#1974] [General]
So i have a file having say {{ cookiecutter.release_tag }} . can this be replaced dynamically in the post-gen hooks in cookie cutter

**Comment by ericof** [comment]
@kabilanmani93 I hope this can help you https://github.com/collective/cookiecutter-plone-starter/blob/main/hooks/post_gen_project.py#L46
